### PR TITLE
Add Boba Network

### PR DESCRIPTION
**Pull request by jet86**
Create basic/initial config file for Boba Network, and include in index.ts

Will add more details and references to the Boba Network config file once they are available.

### CODE REVIEW

**Comment by sz-piotr** [comment]
Hey @jet86. Thank you very much for the PR. There a few things that you should know.
1. You need to provide an icon for the project. It should be placed in `website/src/static/icons` and named `bobanetwork.png`. It also needs to be smaller than 10kb.
2. We are in the final days before releasing a new risk framework. While the items such as `technologyName` and `technologyDetails` and `parameters` are deprecated they are currently still required for our site, because they are what is being displayed. You can either add those items or wait for our new risk framework to be released.
3. For the new risk framework and technology description we require projects to provide references. Please use the `references` field in the various elements of the `technology` object to provide links to documentation, whitepaper or blog posts that describe the issue in detail.

Once those items are resolved we can proceed with a more thorough review of the PR.

**Comment by sz-piotr** [comment]
Any progress on this?

**Comment by jet86** [comment]
Hi @sz-piotr sorry for the delay - we have updated the PR now. Please review and advise if there are any outstanding issues with it.

File: packages/config/src/projects/bobanetwork.ts [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+            text: 'Chain Contracts - Boba documentation',
+            href: 'https://docs.boba.network/developer-docs/chain-contracts',
+          },
+        ],
+      },
+      exitMechanisms: [
+        {
+          name: 'Regular exit',
+          description:
+            'When a user initiates a withdrawal it is processed as a L2 to L1 message. Because Boba Network is an optimistic rollup this transaction has to be included in a block and finalized. This takes several days to happen after which the funds can be withdrawn on L1.',
```
Comment (sz-piotr) [inline packages/config/src/projects/bobanetwork.ts:107]: Any references here?

File: packages/config/src/projects/bobanetwork.ts [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        },
+        {
+          name: 'Fast exit',
+          description:
+            'Users can initiate a fast exit which makes use of liquidity pools, and charges a small fee for the convenience. Users funds can then be withdrawn on L1 after only minutes.',
```
Comment (sz-piotr) [inline packages/config/src/projects/bobanetwork.ts:112]: Any references here? Also this paragraph could mention that the contract holding liquidity for liquidity pools only has a small amount of ETH

File: packages/config/src/projects/bobanetwork.ts [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+        ],
+        references: [
+          {
+            text: 'Execution Contracts - Learn more about the Boba Network execution contracts',
+            href: 'https://docs.boba.network/developer-docs/chain-contracts/execution-contracts',
+          },
+        ],
+      },
+      contracts: {
+        addresses: [
```
Comment (sz-piotr) [inline packages/config/src/projects/bobanetwork.ts:133]: Please include all contracts from this list: https://docs.boba.network/developer-docs/mainnet-addresses

**Review by sz-piotr — CHANGES_REQUESTED** [review]

**Comment by sz-piotr** [comment]
Also, please update the icon to be a 256x256 px image. Thank you.

File: packages/config/src/projects/bobanetwork.ts [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+            href: 'https://docs.boba.network/faq#the-incentive-contract-for-verification-proofs-is-disabled',
+          },
+        ],
+      },
+      dataAvailability: {
+        ...DATA_AVAILABILITY.ON_CHAIN_CANONICAL,
+        references: [
+          {
+            text: 'Data Availability Batches - Paradigm Research',
+            href: 'https://research.paradigm.xyz/optimism#data-availability-batches',
```
[the remark this answers is not in the document]
Comment (jet86) [reply packages/config/src/projects/bobanetwork.ts:81]: Happy to change it to point to our CTC docs if you think that's better, however I want to point out that the Paradigm Research docs on optimistic rollups data availability batches apply equally to Optimism and Boba Network (and note that Optimism's data availability section references this research doc).

File: packages/config/src/projects/bobanetwork.ts [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+
+export const optimism: Project = {
+  name: 'Boba Network',
+  slug: 'bobanetwork',
+  bridges: [
+    {
+      // Proxy__OVM_L1StandardBridge
+      address: '0xdc1664458d2f0B6090bEa60A8793A4E66c2F1c00',
+      sinceBlock: 13012048,
+      tokens: ['ETH', 'USDC', 'OMG', 'DAI', 'WBTC', 'SUSHI', 'UNI', 'LINK', 'DODO', 'REP', 'BAT', 'ZRX'],
```
[the remark this answers is not in the document]
Comment (jet86) [reply packages/config/src/projects/bobanetwork.ts:17]: Note that the other tokens are supported, there just isn't any in there at the moment. I'll commit your suggestion and add PRs as those tokens are added.

File: packages/config/src/projects/bobanetwork.ts [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+    },
+    {
+      // Proxy__L1LiquidityPool
+      address: '0x6b4eab8D55A4f701C3FE58bDb8b3e1f181eA7087',
+      sinceBlock: 13013879,
+      tokens: ['ETH', 'USDC', 'OMG', 'DAI', 'WBTC', 'SUSHI', 'UNI', 'LINK', 'DODO', 'REP', 'BAT', 'ZRX'],
```
[the remark this answers is not in the document]
Comment (jet86) [reply packages/config/src/projects/bobanetwork.ts:23]: As above, note that the other tokens are supported, there just isn't any in there at the moment. I'll commit your suggestion and add PRs as those tokens are added.

File: packages/config/src/projects/bobanetwork.ts [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+  details: {
+    description:
+      'The Ethereum Layer 2: Boba Network. Lower gas, faster, secured by Ethereum.',
+    purpose: 'Universal',
+    links: {
+      websites: ['https://boba.network'],
+      apps: [],
+      documentation: ['https://docs.boba.network/'],
+      explorers: ['https://blockexplorer.boba.network/'],
+      repositories: ['https://github.com/omgnetwork'],
```
[the remark this answers is not in the document]
Comment (jet86) [reply packages/config/src/projects/bobanetwork.ts]: Fixed.

File: packages/config/src/projects/bobanetwork.ts (hunk 1 above)
Comment (jet86) [reply packages/config/src/projects/bobanetwork.ts:107]: Reference added.

File: packages/config/src/projects/bobanetwork.ts (hunk 2 above)
Comment (jet86) [reply packages/config/src/projects/bobanetwork.ts:118]: Reference added. Note that the amount of ETH and other assets will grow as more people onboard and the TVL increases (as is already happening).

File: packages/config/src/projects/bobanetwork.ts (hunk 3 above)
Comment (jet86) [reply packages/config/src/projects/bobanetwork.ts:145]: Added.

File: packages/config/src/projects/bobanetwork.ts [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+    },
+    riskView: {
+      stateValidation: {
+        value: 'Proofs disabled',
+        description:
+          'Currently the system permits invalid state roots. More details in project overview.',
+        sentiment: 'bad',
+      },
+      dataAvailability: RISK_VIEW.DATA_ON_CHAIN,
+      upgradeability: RISK_VIEW.UPGRADABLE_YES,
```
[the remark this answers is not in the document]
Comment (jet86) [reply packages/config/src/projects/bobanetwork.ts:51]: I've confirmed with engineering that:

> vital contracts that have funds stored (optimism + boba) are upgradeable, and there's pause functionality too (all the boba contracts (LPs) have pause, plus a vital optimism contract has too)

**Review by jet86 — COMMENTED** [review]
Should be all good now.

File: packages/config/src/projects/bobanetwork.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (sz-piotr) [reply packages/config/src/projects/bobanetwork.ts:82]: You are actually right. I reexamined the link and including it makes sense here. We'll keep both references.

File: packages/config/src/projects/bobanetwork.ts [hunk 9]
```diff
@@ -0,0 +1,291 @@
...
+              implementation: '0x4828900EB9dB5346dfe33cCf884984D8014C7B4B',
+            },
+          },
+          {
+            name: 'L1 Liquidity Pool',
+            address: '0x6b4eab8D55A4f701C3FE58bDb8b3e1f181eA7087',
+            description: CONTRACTS.UNVERIFIED_DESCRIPTION,
+          },
+          {
+            // TODO: Support this type of proxy
```
Comment (sz-piotr) [inline packages/config/src/projects/bobanetwork.ts:169]: I have added this TODO for my future self. Please ignore it :)

File: packages/config/src/projects/bobanetwork.ts [hunk 10]
```diff
@@ -0,0 +1,291 @@
...
+            name: 'OVM_StateManagerFactory',
+            address: '0x0c4935b421Af8F86698Fb77233e90AbC5f146846',
+          },
+          {
+            name: 'OVM_StateTransitionerFactory',
+            address: '0xc6dd73D427Bf784dd1e2f9F64029a79533ffAb40',
+          },
+          {
+            name: 'OVM_Sequencer',
+            address: '0xE48E5b731FAAb955d147FA954cba19d93Dc03529',
```
Comment (sz-piotr) [inline packages/config/src/projects/bobanetwork.ts:227]: This address does not seem to be a smart contract. Is the address wrong or should we remove this entry?

**Review by sz-piotr — CHANGES_REQUESTED** [review]

File: packages/config/src/projects/bobanetwork.ts (hunk 10 above)
Comment (jet86) [reply packages/config/src/projects/bobanetwork.ts:227]: Oh, that's the sequencer address for Rinkeby. Let me double check the correct address for mainnet and fix it...

Comment (jet86) [reply packages/config/src/projects/bobanetwork.ts]: Updated to the correct mainnet sequencer address now.

**Review by jet86 — COMMENTED** [review]
Updated the incorrect address - let me know if anything else still needs changing.

File: packages/config/src/projects/bobanetwork.ts (hunk 10 above)
Comment (sz-piotr) [reply packages/config/src/projects/bobanetwork.ts]: Thanks. Removed it, because it is not a contract :)

**Review by sz-piotr — APPROVED** [review]
We are good to merge. Waiting for you @jet86 to also say that we are good to go.

**Comment by jet86** [comment]
Yep, good to go. Please merge @sz-piotr 

thanks for all your help.

**Pull request merged**